README.md

```diff
@@ -1,2 +1,29 @@
 # sparta-weekend-hw1
-Min. 3 page website incorporating all learned material so far
+Min. 3 page website incorporating all learned material so far including:
+
+
+# The Task
+
```

Comment:
I would rewrite this to relate to the fact that it's your repo - i.e. "My task was..." or "The brief I was given was..."